core/rfb.js

```diff
@@ -1083,14 +1083,14 @@ export default class RFB extends EventTargetMixin {
 
     _negotiate_ultravnc_sc_auth() {
         if (this._sock.rQwait("UltraVNC Single Click auth msg size", 4)) {return false;}
-        const strlen = this._sock.rQshift32L()
+        const strlen = this._sock.rQshift32L();
 
         if (this._sock.rQwait("UltraVNC Single Click auth msg body", strlen, 4)) {return false;}
         this._rfb_ultravnc_scinfo = this._sock.rQshiftStr(strlen);
 
-        Log.Info(this._rfb_ultravnc_scinfo)
+        Log.Info(this._rfb_ultravnc_scinfo);
 
-        this._sock.send([1,0,0,0]);
+        this._sock.send([1, 0, 0, 0]);
```

Comment:
Please merge these fixes with the original commit.

Reply:
Sorry,I don't know much about github's pull request, so I don't quite understand what you mean.

Reply:
Does it mean rebase?

Reply:
And then do a force push (`git push -f`) to update the branch and automatically update this pull request.

Reply:
Thank you for your help.